Clean Validation Environment workflow fails

Running Clean Validation Environment workflow from templated AzureTRE-Deployment, experiencing the following error:

[Clean](https://github.com/daemel/AzureTRE-P2/actions/runs/3092183217/jobs/5003161177#step:3:8)
Az CLI Login failed. Please check the credentials and make sure az is installed on the runner. For more information refer https://aka.ms/create-secrets-for-GitHub-workflows

Doing so will allow login to work, unfortunately the clean ci validation script is not part of the Deployment Repository.  


Log output 
Run devops/scripts/clean_ci_validation_envs.sh
  devops/scripts/clean_ci_validation_envs.sh
  shell: /usr/bin/bash -e ***0***
  env:
    AZURE_HTTP_USER_AGENT: 
    AZUREPS_HOST_ENVIRONMENT: 
    GITHUB_TOKEN: ***
    MAIN_TRE_ID: ***
    BRANCH_LAST_ACTIVITY_IN_HOURS_FOR_STOP: 2
    BRANCH_LAST_ACTIVITY_IN_HOURS_FOR_DESTROY: 48
/home/runner/work/_temp/40f733cb-9622-4df[1](https://github.com/daemel/AzureTRE-P2/actions/runs/3106173851/jobs/5032712941#step:4:1)-bec1-589c79a43466.sh: line 1: devops/scripts/clean_ci_validation_envs.sh: No such file or directory
Error: Process completed with exit code 1[2](https://github.com/daemel/AzureTRE-P2/actions/runs/3106173851/jobs/5032712941#step:4:2)[7](https://github.com/daemel/AzureTRE-P2/actions/runs/3106173851/jobs/5032712941#step:4:7).

Tested after latest PR update - same result 

Run devops/scripts/clean_ci_validation_envs.sh
  devops/scripts/clean_ci_validation_envs.sh
  shell: /usr/bin/bash -e ***0***
  env:
    AZURE_HTTP_USER_AGENT: 
    AZUREPS_HOST_ENVIRONMENT: 
    GITHUB_TOKEN: ***
    MAIN_TRE_ID: ***
    BRANCH_LAST_ACTIVITY_IN_HOURS_FOR_STOP: [2](https://github.com/daemel/AzureTRE-P2/actions/runs/3113897988/jobs/5049096569#step:4:2)
    BRANCH_LAST_ACTIVITY_IN_HOURS_FOR_DESTROY: 48
/home/runner/work/_temp/61d7a695-3fa3-4778-a6ae-091a7412e7ea.sh: line 1: devops/scripts/clean_ci_validation_envs.sh: No such file or directory
Error: Process completed with exit code 12[7](https://github.com/daemel/AzureTRE-P2/actions/runs/3113897988/jobs/5049096569#step:4:7).

--- a/.github/workflows/clean_validation_envs.yml
+++ b/.github/workflows/clean_validation_envs.yml
@@ -31,4 +31,4 @@ jobs:
           MAIN_TRE_ID: ${{ secrets.TRE_ID }}
           BRANCH_LAST_ACTIVITY_IN_HOURS_FOR_STOP: 2
           BRANCH_LAST_ACTIVITY_IN_HOURS_FOR_DESTROY: 48
-        run: devops/scripts/clean_ci_validation_envs.sh
+        run: .github/scripts/clean_ci_validation_envs.sh

